src/Blocks/utils.jl

```diff
@@ -10,9 +10,8 @@
             input = true,
             description = "Inner variable in RealInput $name"
         ]
-        u = collect(u)
     end
-    ODESystem(Equation[], t, [u...], []; name = name, guesses = [u => u_start])
+    ODESystem(Equation[], t, [u], []; name = name, guesses = [u => u_start])
```

Comment:
If I as a user pass `u_start=1` I want that to be true, not only a guess. I wonder if these `u_start` are needed at all?

Reply:
You should be able to change the guess. Why would this be a direct value instead of a guess? The value is set by the variable that defines the input and the function that defines the output, any default here would just be an extra equation causing an overconstrained system. Indeed if someone sets a default then it should be true, but why would it be normal to set an overconstrained system with inputs and outputs? We can make that another kwarg that is nothing by default?

Reply:
it should probably just be renamed `guess` and deprecate the old name.

Reply:
that would be much less confusing

Reply:
I'd prefer if we stop using the word "default" as well in favor of either "start" or "initial", but that's a discussion for another time and place :P

Reply:
Fixed in https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/291/commits/7b38a539079bad5125367c881de459db46868673